src/main/java/io/vertx/core/net/ClientOptionsBase.java

```diff
@@ -167,6 +170,25 @@ public ProxyOptions getProxyOptions() {
     return proxyOptions;
   }
 
+  /**
+   * @return the local interface to bind for network connections.
+   */
+  public String getLocalAddress() {
+    return localAddress;
+  }
+
+  /**
+   * Set the local interface to bind for network connections. When the local address is null,
+   * it will pick any local address, the default local address is null.
```

Comment:
I can't think of a better wording, but strictly speaking it's not "any" address, it an address which has a route to the destination. Maybe it's not important to get into such details, still wanted to let you know.